testing/iprutils/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: Breno Leitao <breno.leitao@gmail.com>
+pkgname=iprutils
+pkgver=2.4.14
+pkgrel=0
+pkgdesc="user space tools required by the ipr device driver."
+url="https://github.com/bjking1/iprutils"
+arch="all"
+license="CPL"
+depends="automake libtool m4 autoconf ncurses-dev zlib-dev"
+makedepends=""
+install=
+subpackages="$pkgname-doc"
+_pkgver=$(echo $pkgver | sed 's/\./-/g')
```

Comment:
We try to avoid subshells in main context. This is to do with the build scripting performance.
Wouldn't something like `${pkgver//./-}` work ?